api/refresh_continuous_aggregate.md

```diff
@@ -26,10 +26,17 @@ refreshing in order to align with bucket boundaries.
 ### Required arguments
 
 |Name|Type|Description|
-|---|---|---|
-| `continuous_aggregate` | REGCLASS | The continuous aggregate to refresh. |
-| `window_start` | INTERVAL | Start of the window to refresh, has to be before `window_end`. `NULL` is eqivalent to `MIN(timestamp)` of the hypertable. |
-| `window_end` | INTERVAL | End of the window to refresh, has to be after `window_start`. `NULL` is eqivalent to `MAX(timestamp)` of the hypertable. |
+|-|-|-|
+|`continuous_aggregate`|REGCLASS|The continuous aggregate to refresh.|
+|`window_start`|INTERVAL|Start of the window to refresh, has to be before `window_end`. `NULL` is eqivalent to `MIN(timestamp)` of the hypertable.|
+|`window_end`|INTERVAL|End of the window to refresh, has to be after `window_start`. `NULL` is eqivalent to `MAX(timestamp)` of the hypertable.|
+
+<highlight type="warning">
```

Comment:
This warning applies only to the policy. It is not needed for the refresh_continuous_aggregate case. There are cases where it is valid to call this function with NULL for window_end. Could you remove this warning from this function?

Reply:
Done! Is it good otherwise, or are there other changes I should make?